kapre/time_frequency.py

```diff
@@ -156,8 +159,12 @@ def call(self, x):
             waveforms = tf.pad(
                 waveforms, tf.constant([[0, 0], [0, 0], [int(self.n_fft - self.hop_length), 0]])
             )
+        
+        stft_function = tf.signal.stft
```

Comment:
Can we make it even more straightforward like..
```
if self.use_parallel_stft:
    stft_function = parallel_stft
else:
    stft_function = tf.signal.stft
```
? 

Or an one-liner like `stft_function = parallel_stft if self.use_parallel_stft else tf.signal.stft`